tests/test_population.py

```diff
@@ -122,6 +122,18 @@ def test_breed_works_with_kwargs(self, simple_chromosomes, simple_evaluation_fun
         assert len(pop2) == 400
         assert pop2.intended_size == 400
 
+    def test_breed_raises_with_multiple_values_for_kwarg(self, simple_population):
+
+        def dummy_combiner(x, y):
```

Comment:
why not use a `lambda`?

Reply:
That would work just a well. In this case I find any tracebacks easier to read but I guess that isn't really a problem here.